File: sdkwork-app-sdk-go/types/browse_history_query_form.go
package types


type BrowseHistoryQueryForm struct {
	Type string `json:"type"`
	TargetId string `json:"targetId"`
	Page int `json:"page"`
	Size int `json:"size"`
	SortField string `json:"sortField"`
	SortOrder string `json:"sortOrder"`
}

File: sdkwork-app-sdk-csharp/Models/BrowseHistoryQueryForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class BrowseHistoryQueryForm
    {
        public string? Type { get; set; }
        public string? TargetId { get; set; }
        public int? Page { get; set; }
        public int? Size { get; set; }
        public string? SortField { get; set; }
        public string? SortOrder { get; set; }
    }
}
